flask_smorest/blueprint.py

```diff
@@ -162,6 +165,32 @@ def decorator(func):
 
         return decorator
 
+    def register_blueprint(self, blueprint, **options):
+        """Register a nested blueprint in application
+
+        Also stores doc info from the nested bluepint for later registration.
+
+        Use this to register a nested :class:`Blueprint <Blueprint>`.
+
+        :param Blueprint blueprint: Blueprint to register under this blueprint.
+        :param options: Options to be forwarded to the underlying
+            :meth:`flask.Blueprint.register_blueprint` method.
+
+        See :ref:`register-nested-blueprints`.
+        """
+        blp_name = options.get("name", blueprint.name)
+
+        # Inherit all endpoints
+        self._docs.update(
+            {
+                ".".join((blp_name, endpoint_name)): doc
+                for endpoint_name, doc in blueprint._docs.items()
+            }
+        )
+        self._endpoints.extend(blueprint._endpoints)
```

Comment:
```py
self._endpoints.extend(blueprint._endpoints)
```

I'm not sure this line is actually useful, is it?

This is only used inside each Blueprint to avoid having multiple routes with the same endpoint name in `self._docs`. By the time we pass here, `self._docs` is already written (and namespaced for nested blueprints thanks to the code you added above).

Reply:
You're right. Removed this line